content/en/docs/components/pipelines/v2/author-a-pipeline/pipelines.md

```diff
@@ -115,11 +115,31 @@ def my_pipeline():
     with dsl.ExitHandler(exit_task=print_status_task):
         fail_op()
 ```
+#### Ignore upstream failure
+The [ignore_upstream_failure()][ignore-upstream-failure] configuration from [pipeline_task] is useful if the caller task has upstream dependencies. If the pipeline task fails, this method converts the caller task into an exit handler and continues to collect outputs from the upstream tasks.
```

Comment:
While it is a true statement, I wonder if there's more specific way to characterize the purpose `.ignore_upstream_failure` in the first sentence.

What about "...is useful if the caller task wishes to ignore the failures of upstream tasks?"

cc @sen-sam